cmd/util.go

```diff
@@ -45,7 +44,6 @@ func newUtilCommand() *cobra.Command {
 	flags.AuthOIDCClaimUsername(generateJWTCmd.Flags())
 	flags.AuthOIDCClaimPermissions(generateJWTCmd.Flags())
 	flags.AuthOIDCClaimRoles(generateJWTCmd.Flags())
-	flags.AuthOIDCCertKeyFile(generateJWTCmd.Flags())
```

Comment:
why are you removing this? How can your e2e work? You should only remove the password